packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,386 @@
+    <View style={styles.descriptionContainer}>
+      <Text style={{ ...fontStyles.h1, flex: 1 }}>{message}</Text>
+      <View style={styles.descriptionImage}>
+        <DappsExplorerLogo />
+      </View>
+    </View>
+  )
+}
+
+function CategoryHeader({ category }: { category: CategoryWithDapps }) {
+  return (
+    <View style={styles.categoryContainer}>
+      <View
+        style={{
+          ...styles.categoryTextContainer,
+          backgroundColor: category.backgroundColor,
+        }}
+      >
+        <Text style={{ ...styles.categoryText, color: category.fontColor }}>{category.name}</Text>
+      </View>
+    </View>
+  )
+}
+
+function Dapp({ dapp, onPressDapp }: DappProps) {
+  const onPress = () => onPressDapp(dapp)
+
+  return (
+    <Card style={styles.card} rounded={true} shadow={Shadow.Soft}>
+      <TouchableOpacity style={styles.pressableCard} onPress={onPress}>
+        <Image source={{ uri: dapp.iconUrl }} style={styles.dappIcon} />
+        <View style={styles.itemTextContainer}>
+          <Text style={styles.itemTitleText}>{dapp.name}</Text>
+          <Text style={styles.itemSubtitleText}>{dapp.description}</Text>
+        </View>
+        <LinkArrow style={styles.linkArrow} />
+      </TouchableOpacity>
+    </Card>
+  )
+}
+
+const styles = StyleSheet.create({
+  safeAreaContainer: {
+    flex: 1,
+  },
+  centerContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    flex: 1,
+  },
+  categoryContainer: {
+    alignItems: 'flex-start',
+    justifyContent: 'center',
+    flex: 1,
+    flexDirection: 'column',
+    marginTop: SECTION_HEADER_MARGIN_TOP,
+  },
+  itemTextContainer: {
+    flex: 1,
+  },
+  descriptionContainer: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    flex: 1,
+    marginHorizontal: Spacing.Smallest8,
+  },
+  helpIconContainer: {
+    padding: variables.contentPadding,
+  },
+  loadingIcon: {
+    marginVertical: Spacing.Thick24,
+    height: 108,
+    width: 108,
+  },
+  dappIcon: {
+    width: 40,
+    height: 40,
+    borderRadius: 8,
+    marginRight: Spacing.Regular16,
+  },
+  pressableCard: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    flex: 1,
+  },
+  card: {
+    marginTop: Spacing.Regular16,
+    flex: 1,
+    alignItems: 'center',
+  },
+  categoryTextContainer: {
+    borderRadius: 100,
+  },
+  // Padding values honor figma designs
+  categoryText: {
+    ...fontStyles.regular,
+    fontSize: 12.5,
+    borderRadius: 100,
+    paddingHorizontal: 11,
+    paddingVertical: 4,
+  },
+  itemTitleText: {
+    ...fontStyles.small,
+    color: Colors.dark,
+  },
+  itemSubtitleText: {
+    ...fontStyles.small,
+    color: Colors.gray5,
+  },
+  linkArrow: {
+    height: 20,
+    width: 20,
+    marginLeft: Spacing.Regular16,
+  },
+  descriptionImage: {
+    height: 106,
+    width: 94,
+    marginLeft: Spacing.Smallest8,
+  },
+  bottomSheetHeader: {
+    flex: 1,
+    flexDirection: 'row-reverse',
+    alignItems: 'flex-start',
```

Comment:
super nit: i think you can have `alignItems: 'flex-end'` and remove the flexDirection to achieve the same effect?

Reply:
actually you could even just add `style={{ alignSelf: 'flex-end' }}` to the `TopBarIconButton` and remove the need for the surrounding `View` too

Reply:
I think you are right, let me try it. Thanks!